lms/djangoapps/courseware/tests/test_group_access.py

```diff
@@ -53,7 +55,7 @@ def _wrapper(self, *args):  # pylint: disable=missing-docstring
 
 @attr('shard_2')
 @ddt.ddt
-class GroupAccessTestCase(ModuleStoreTestCase):
+class GroupAccessTestCase(ModuleStoreTestCase, MilestonesTestCaseMixin):
```

Comment:
This test case shouldn't require milestones, right?  And the milestones helper method that is called in access.py is a no-op if `MILESTONES_APP` is `False`.